nixos/modules/profiles/macos-builder.nix

```diff
@@ -15,6 +20,19 @@ in
   imports = [
     ../virtualisation/qemu-vm.nix
 
+    (lib.mkRenamedOptionModuleWith {
+      sinceRelease = 2411;
```

Comment:
While I'm not a big fan of it, some of these `darwin-builder` options are effectively aliases grouped together for UX purposes. For consistency, could leave this option alone and make it set `virtualisation.diskSize`, following the other options as examples.
@Gabriella439, author of the module, wdyt? Should we standardize on generic options for these?
It would mesh with the generalization in https://github.com/NixOS/nixpkgs/pull/347255

Reply:
Left the option in linux-vm-builder as-is after a rebase.

virtualisation.diskSize is already set there, so I don't think we need additional changes here?